src/main/java/org/fundacion/jala/converter/models/AuthenticationRequest.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package org.fundacion.jala.converter.models;
+
+public class AuthenticationRequest {
+    private String username;
+    private String password;
+
+    public AuthenticationRequest() {
```

Comment:
Maybe you should delete this constructor

Reply:
I deleted the constructor.